Lib/idlelib/pyparse.py

```diff
@@ -50,15 +50,15 @@
 # Match any flavor of string; the terminating quote is optional
 # so that we're robust in the face of incomplete program text.
 
-_match_stringre = re.compile(r"""
-    \""" [^"\\]* (?:
+_match_stringre = re.compile(r'''
+    """ [^"\\]* (?:
                      (?: \\. | "(?!"") )
                      [^"\\]*
                  )*
-    (?: \""" )?
+    (?: """ )?
 
 |   " [^"\\\n]* (?: \\. [^"\\\n]* )* "?
-
+''' r"""
```

Comment:
Glenn proposed also other method: replace `"""` and `'''` in REs with `"{3}` and `'{3}` correspondingly.

You can also inserts a backslash in the middle of triple quotes: `"\""` or `""\"`.